### PR TITLE
add cosh to jax

### PR DESCRIPTION
**Pull request by PatricYan**
- [ ] #3112 
-  add cosh to jax

### CODE REVIEW

**Comment by PatricYan** [comment]
Hi, would you like to make a review for the PR? I have resolved the conflict.

**Comment by PatricYan** [comment]
Hi @Auxia , would you like to make a review for the PR?

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_lax_operators.py [hunk 1]
```diff
@@ -1127,3 +1127,33 @@ def test_jax_lax_ge(
         x=xs[0],
         y=xs[1],
     )
+
+
+# cosh
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=ivy_jax.valid_float_dtypes),
```
Comment (Auxia) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_lax_operators.py:1135]: Please also add:
`as_variable=st.booleans()`
`native_array=st.booleans()`
These are required inputs for tests. Check the [docs](https://lets-unify.ai/ivy/deep_dive/17_ivy_frontends_tests.html#ivy-tan) for further explanation

**Review by Auxia — CHANGES_REQUESTED** [review]
Hi! Thanks for contributing to Ivy. Please make these slight changes and we should be good to merge this PR.

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_lax_operators.py (hunk 1 above)
Comment (PatricYan) [reply ivy_tests/test_ivy/test_frontends/test_jax/test_jax_lax_operators.py:1135]: Hi @Auxia , it don't need to add 
```
as_variable=st.booleans()
native_array=st.booleans()
```
 by adding **@handle_cmd_line_args**, would you like to make a check for it?

**Comment by Auxia** [comment]
Hi, I did some further studying and yes, your point is valid. Thanks for contributing to Ivy! I will merge this PR now.

**Pull request merged**